### PR TITLE
Added stop for music player instance

### PR DESCRIPTION
**Pull request by maxminoS**
Also bound the stop button to the stop media key

### CODE REVIEW

File: desktop-environment.el [hunk 1]
```diff
@@ -487,6 +491,12 @@ the command `desktop-environment-screenshot-partial-command'."
   (message "%s"
            (desktop-environment--shell-command-to-string desktop-environment-music-next-command)))
 
+(defun desktop-environment-music-stop()
```
Comment (DamienCassou) [inline desktop-environment.el:494]: can you please add a space before the list of arguments?

**Review by DamienCassou — CHANGES_REQUESTED** [review]
Excellent. Would you mind adding the necessary key bindings to the README please?

**Comment by maxminoS** [comment]
Sure!

I have added the relevant keybindings into the README and fixed the function-argument spacing.

**Review by DamienCassou — APPROVED** [review]
This is perfect, thank you. Final step is to squash all your commits into just one.

**Comment by DamienCassou** [comment]
Please also rebase on top of latest master, your branch is out-of-date.

**Comment by maxminoS** [comment]
I think I've just successfully rebased it and will just need the review to merge it.

**Review by DamienCassou — CHANGES_REQUESTED** [review]
Please squash your 2 commits into just 1 please (you did a "merge" instead of a "rebase").

**Comment by maxminoS** [comment]
Sorry for the mistake, I definitely checked the logs this time and made sure I rebased it correctly.

**Review by DamienCassou — APPROVED** [review]
Perfect! Thank you very much

**Pull request merged**